app/app.go

```diff
@@ -340,6 +342,18 @@ func (app *Application) GetChainBondDenom() string {
 	return "stake"
 }
 
+func (app *Application) GetFeeDenomsWhitelist() []string {
+	chainID := app.ChainID()
+
+	if strings.HasPrefix(chainID, "core-") {
+		return FeeDenomsWhitelistMainnet
+	} else if strings.HasPrefix(chainID, "test-core-") {
+		return FeeDenomsWhitelistTestnet
+	}
+
+	return []string{"stake"}
```

Comment:
can you for random chain do something like allow all?  special condition? Not for this release but in general..

Reply:
sure 👍